.github/workflows/vcpkg_ci_amd64.yml

```diff
@@ -207,7 +207,7 @@ jobs:
       - name: 'Remill build'
         shell: 'bash'
         working-directory: remill
-        if: contains(matrix.llvm, 'llvm-15')
+        # if: contains(matrix.llvm, 'llvm-15')
```

Comment:
should we just remove these instead of commenting them out?

Reply:
Yeah, I'll remove it. Better to be clean than to save a little bit of time when next LLVM comes out 👍